wikimedia/wikipedia-ios
# Image recommendations - Localization updates

**Phabricator:** https://phabricator.wikimedia.org/T363925#9765667

### Notes
* Updates the localization of the Edit Save view to match content language

### Test Steps
1. On the edit save view controller, make sure all the copy matches the content language
